libs/common-services/src/test/java/uk/gov/di/ipv/core/library/gpg45/Gpg45ScoresTest.java

```diff
@@ -69,6 +71,10 @@ void calculateGpg45ScoresRequiredToMeetAProfile() {
                 Arrays.asList(new Gpg45Scores(EV_32, 0, 0, 2)),
                 new Gpg45Scores(EV_22, 1, 2, 0)
                         .calculateGpg45ScoresRequiredToMeetAProfile(Arrays.asList(M1B)));
+        assertEquals(
+                Arrays.asList(new Gpg45Scores(EV_22, 0, 1, 1)),
+                new Gpg45Scores(EV_00, 0, 0, 0)
```

Comment:
In the F2F scenario we'll have fraud and activity score - so for thin file similar to above eg. `new Gpg45Scores(List.of(), 0, 1, 0)`